package program;

public class Exceptionhandling {

	public static void main(String[] args) {
	/*	try {	
		int c=1/1;
		
		
		
System.out.println( c);
	}
		
		catch( ArithmeticException a)
		{     
			System.out.println("Print catch");
		
		
		}*/
		
		int c=1/1;
		}

}
